### ISSUE TITLE
Airtime schedule blank page

### ISSUE THREAD

**Issue by xabispacebiker** [LibreTime/libretime#492]
Suddenly, the schedule shows a blank page. zend.php is logging the following error:

```
2018-06-13T15:02:49+02:00 ERR (3):  [Logging.php:166 - loggingShutdownCallback()] - Call to a member function getDbArtistName() on null at /usr/share/airtime/php/airtime_mvc/application/models/Schedule.php:272
array (
  'type' => 1,
  'message' => 'Call to a member function getDbArtistName() on null',
  'file' => '/usr/share/airtime/php/airtime_mvc/application/models/Schedule.php',
  'line' => 272,
)
```

I did some modifications to Schedule.php line 272:

```
                    $previousMediaName = $previousFile->getDbArtistName() . " - " . $previousFile->getDbTrackTitle();
                    $previousMetadata = CcFiles::sanitizeResponse($previousFile);
```

to 

```
                if ($previousFile)
                {
                    $previousMediaName = $previousFile->getDbArtistName() . " - " . $previousFile->getDbTrackTitle();
                    $previousMetadata = CcFiles::sanitizeResponse($previousFile);
                }
```

and now it is working

**Comment by hairmare** [comment]
Do you have any idea what should have been in `$previousFile`? The schedule can be a bit fragile in places so maybe adding your change as a pr might even make sense.

**Comment by Robbt** [comment]
This was likely fixed with #803
